recipes/gmp/all/conandata.yml

```
@@ -1,7 +1,7 @@
 sources:
   "6.2.1":
-    url: "https://gmplib.org/download/gmp/gmp-6.2.1.tar.bz2"
-    sha256: "eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c"
+    url: "https://github.com/alisw/GMP/archive/refs/tags/v6.2.1.tar.gz"
```

would it be reasonable to use the mirror feature of the URL?

```suggestion
    url: 
      - "https://github.com/alisw/GMP/archive/refs/tags/v6.2.1.tar.gz"
      - "https://gmplib.org/download/gmp/gmp-6.2.1.tar.bz2"
```